shared/src/main/kotlin/com/egm/stellio/shared/util/JsonUtils.kt

```diff
@@ -51,3 +52,9 @@ object JsonUtils {
         return mapperWithMixin.writer(filterProvider).writeValueAsString(input)
     }
 }
+
+fun String.toUri(): URI =
```

Comment:
URI is not really related to Json, can we move it to another Utils class or object ?

Reply:
Indeed, any suggestion ?

Reply:
StringUtils ?

Reply:
@bobeal

Reply:
URIUtils (in shared.util)?